charts/flink-operator-crd/templates/flink-operator-crd.yaml

```diff
@@ -0,0 +1,6378 @@
+# This is originally from https://github.com/spotify/flink-on-k8s-operator/releases/download/v0.3.0/flink-operator.yaml
+# with modifications for general usage
+#
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: unapproved
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/flink-operator-serving-cert
+    controller-gen.kubebuilder.io/version: v0.6.2
+  name: flinkclusters.flinkoperator.k8s.io
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          name: flink-operator-webhook-service
+          namespace: {{ .Release.Namespace }}
```

Comment:
This file does not get templatized ... remove all the references to `{{ .Release.Namespace }}`

Reply:
@diranged should we use a specific namespace, or this field is not needed? (it was `flink-operator-system` originally)

Reply:
I believe we should yank out or comment out that section.

Reply:
namespace is a required field, so commenting out webhook conversion section https://v1-20.docs.kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion

Reply:
Looks like we need this webhood service though? @diranged

Reply:
Verified locally with this section commented out and had no issue.